web/modules/custom/dept_migrate/modules/dept_migrate_group_files/config/install/migrate_plus.migration.d7_file.yml

```diff
@@ -10,13 +10,12 @@ migration_tags:
 migration_group: migrate_group_files
 label: 'Public files'
 source:
-  plugin: d7_file
+  plugin: d7_file_uuid
```

Comment:
Needed because the d7_file plugin uses D7 fid as the primary id. We want the UUID instead to avoid clashes.